### DISCUSSION TITLE
Trouble with dnsmasq - port 53 in use

### DISCUSSION THREAD

**Discussion by pilondav** [iiab/iiab#3824] [Q&A]
Hi, I'm installing the latest version of IIAB on a Raspberry Pi 4 but I'm having trouble with DNS.  I started with a clean load of the latest version of RasPi OS. Browsing to http://box.local returns a "403 Forbidden" error.  

I've run "sudo iiab" and "sudo iiab-network" several times, with reboots, and nothing changes.  Running "sudo dnsmasq" returns "dnsmasq: failed to create listening socket for port 53: Address already in use".  I can't figure out what service might already be using port 53.  

Has anyone run into this issue?

Thanks,
Dave

**Comment by holta** [comment]
Hi @pilondav

Can you please run `sudo iiab-diagnostics` and post the URL that results?
